tests/functional/spec/features/calculated_summary/new_calculated_summary_repeating_and_static_answers.spec.js

```diff
@@ -0,0 +1,200 @@
+    );
+    await assertSummaryValues(["£300.00", "£200.00", "£100.00", "£30.00", "£15.00", "£10.00", "£5.00", "£0.00"]);
+
+    // second calculated summary
+    await $(CalculatedSummarySpendingPage.submit()).click();
+    await expect(await $(CalculatedSummaryVisitsPage.calculatedSummaryTitle()).getText()).to.contain(
+      "We calculate the total visits to the shop to be 14. Is this correct?"
+    );
+    await assertSummaryValues(["4", "3", "2"]);
+    await $(CalculatedSummaryVisitsPage.submit()).click();
+    await expect(await browser.getUrl()).to.contain(SummaryPage.pageName);
+  });
+
+  it("Given I remove an item from the list which changes the calculated summaries, I return to each calculated summary to confirm new total with answers removed", async () => {
+    await expect(await $(SummaryPage.supermarketsListLabel(1)).getText()).to.equal("Tesco");
+    await expect(await $(SummaryPage.supermarketsListLabel(2)).getText()).to.equal("Lidl");
+    await expect(await $(SummaryPage.supermarketsListLabel(3)).getText()).to.equal("Sainsburys");
+    await expect(await $(SummaryPage.supermarketsListLabel(4)).isExisting()).to.be.false;
+    await $(SummaryPage.supermarketsListRemoveLink(1)).click();
+
+    await expect(await browser.getUrl()).to.contain(ListCollectorRemovePage.pageName);
+    await $(ListCollectorRemovePage.yes()).click();
+    await $(ListCollectorRemovePage.submit()).click();
+
+    // section is now incomplete so step through each block until calculated summary is re-confirmed
+    await expect(await browser.getUrl()).to.contain(DynamicAnswerPage.pageName);
+    await $(DynamicAnswerPage.submit()).click();
+    await $(ExtraSpendingBlockPage.submit()).click();
+    await $(ExtraSpendingMethodBlockPage.submit()).click();
+
+    // Tesco is now gone
+    await expect(await $(CalculatedSummarySpendingPage.calculatedSummaryTitle()).getText()).to.contain(
+      "We calculate the total cost of your weekly shopping to be £380.00. Is this correct?"
+    );
+    await assertSummaryValues(["£200.00", "£100.00", "£15.00", "£10.00", "£5.00", "£50.00"]);
+    await $(CalculatedSummarySpendingPage.submit()).click();
+    await expect(await $(CalculatedSummaryVisitsPage.calculatedSummaryTitle()).getText()).to.contain(
+      "We calculate the total visits to the shop to be 10. Is this correct?"
+    );
+    await assertSummaryValues(["3", "7"]);
+    await $(CalculatedSummaryVisitsPage.submit()).click();
+
+    await expect(await $(SummaryPage.supermarketsListLabel(1)).getText()).to.equal("Lidl");
+    await expect(await $(SummaryPage.supermarketsListLabel(2)).getText()).to.equal("Sainsburys");
+    await expect(await $(SummaryPage.supermarketsListLabel(3)).isExisting()).to.be.false;
+  });
+
+  it("Given I proceed to the second section, the calculated summary of days I visit the supermarket is used as a maximum to the first question", async () => {
```

Comment:
Maybe worth adding an additional test to cover the status of the section being partially complete if you complete this section but go back and change the answer to the used as a maximum?

Reply:
done 👍